engine/table/src/main/java/io/deephaven/engine/util/GroovyDeephavenSession.java

```diff
@@ -288,7 +384,7 @@ private static Class<?> loadClass(String className) throws ClassNotFoundExceptio
         }
     }
 
-    private static boolean classExists(String className) {
+    private boolean classExists(String className) {
```

Comment:
Reviewer note: these could be made static again if the classloader was passed as an argument, it is needed in `loadClass`.